---
layout: page
title: Publications
subtitle: Peer-reviewed papers and scientific output   
---
You can also browse my [Google Scholar](https://scholar.google.com/citations?user=us0AM-wAAAAJ&hl=en).

### 2025

### 2024
[Rapid homologue juxtaposition during meiotic chromosome pairing](https://www.nature.com/articles/s41586-024-07999-5)  
T Nozaki, B Weiner, N Kleckner  
Nature 634 (8036), 1221-1228

### 2023
[Condensed but liquid-like domain organization of active chromatin regions in living human cells](https://www.science.org/doi/full/10.1126/sciadv.adf1488)  
T Nozaki, S Shinkai, S Ide, K Higashi, S Tamura, MA Shimazoe, ...  
Science Advances 9 (14), eadf1488

[海外だより~ Science is fun.~](https://www.jstage.jst.go.jp/article/biophys/63/6/63_340/_article/-char/ja/)  
T NOZAKI  
生物物理 63 (6), 340-341

### 2021
[High Temporal Resolution 3D Live-Cell Imaging of Budding Yeast Meiosis Defines Discontinuous Actin/Telomere-Mediated Chromosome Motion, Correlated Nuclear Envelope Deformation and Actin Filament Dynamics](https://www.frontiersin.org/journals/cell-and-developmental-biology/articles/10.3389/fcell.2021.687132/full)  
T Nozaki, F Chang, B Weiner, N Kleckner
Frontiers in Cell and Developmental Biology 9, 687132

### 2020
[Advances in Chromatin and Chromosome Research: Perspectives from Multiple Fields](https://www.sciencedirect.com/science/article/pii/S109727652030469X)  
AA Agbleke, A Amitai, JD Buenrostro, A Chakrabarti, L Chu, AS Hansen, ...
Molecular cell 79 (6), 881-901

### 2019
[Organization of fast and slow chromatin revealed by single-nucleosome dynamics](https://www.pnas.org/doi/full/10.1073/pnas.1907342116)  
SS Ashwin, T Nozaki, K Maeshima, M Sasai
Proceedings of the National Academy of Sciences 116 (40), 19939-19944

[Single nucleosome imaging reveals loose genome chromatin networks via active RNA polymerase II](https://rupress.org/JCB/article/218/5/1511/120924)  
R Nagashima, K Hibino, SS Ashwin, M Babokhov, S Fujishiro, R Imai, ...
Journal of Cell Biology 218 (5), 1511-1530

### 2018
[Dynamic chromatin folding in the cell](https://www.sciencedirect.com/science/article/abs/pii/B9780128034804000041)  
T Nozaki, DF Hudson, S Tamura, K Maeshima
Nuclear architecture and dynamics, 101-122

### 2017
[Density imaging of heterochromatin in live cells using orientation-independent-DIC microscopy](https://www.molbiolcell.org/doi/full/10.1091/mbc.e17-06-0359)  
R Imai, T Nozaki, T Tani, K Kaizu, K Hibino, S Ide, S Tamura, K Takahashi, ...
Molecular biology of the cell 28 (23), 3349-3359

[Dynamic organization of chromatin domains revealed by super-resolution live-cell imaging](https://www.cell.com/molecular-cell/fulltext/S1097-2765(17)30445-8)  
T Nozaki, R Imai, M Tanbo, R Nagashima, S Tamura, T Tani, Y Joti, ...
Molecular cell 67 (2), 282-293. e7

[Bridging the dynamics and organization of chromatin domains by mathematical modeling](https://www.tandfonline.com/doi/full/10.1080/19491034.2017.1313937)  
S Shinkai, T Nozaki, K Maeshima, Y Togashi
Nucleus 8 (4), 353-359

### 2016
[Dynamic nucleosome movement provides structural information of topological chromatin domains in living human cells](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005136)  
S Shinkai, T Nozaki, K Maeshima, Y Togashi
PLoS computational biology 12 (10), e1005136

### 2015
[The physical size of transcription factors is key to transcriptional regulation in chromatin domains](https://iopscience.iop.org/article/10.1088/0953-8984/27/6/064116/meta)  
K Maeshima, K Kaizu, S Tamura, T Nozaki, T Kokubo, K Takahashi
Journal of Physics: Condensed Matter 27 (6), 064116

### 2014
[Chromatin as dynamic 10-nm fibers](https://link.springer.com/article/10.1007/s00412-014-0460-2)  
K Maeshima, R Imai, S Tamura, T Nozaki
Chromosoma 123 (3), 225-237

[Genome dynamics revealed by single-molecule imaging and computer simulation](https://europepmc.org/article/med/24864446)  
R Imai, K Kaizu, T Nozaki, K Maeshimau, K Takahashi
Seikagaku. The Journal of Japanese Biochemical Society 86 (2), 192-200

[Single nucleosome imaging in living human cells](https://www.jstage.jst.go.jp/article/cytologia/79/1/79_1/_article/-char/ja/)  
T Nozaki, R Imai, S Tamura, K Maeshima
Cytologia 79 (1), 1-1

### 2013
[Flexible and dynamic nucleosome fiber in living mammalian cells](https://www.tandfonline.com/doi/full/10.4161/nucl.26053)  
T Nozaki, K Kaizu, CG Pack, S Tamura, T Tani, S Hihara, T Nagai, ...
Nucleus 4 (5), 349-356

[Computational analysis of associations between alternative splicing and histone modifications](https://www.sciencedirect.com/science/article/pii/S001457931300063X)  
Y Shindo, T Nozaki, R Saito, M Tomita
FEBS letters 587 (5), 516-521

### 2012
[Local nucleosome dynamics facilitate chromatin accessibility in living mammalian cells](https://www.cell.com/cell-reports/fulltext/S2211-1247(12)00389-0?script=true)  
S Hihara, CG Pack, K Kaizu, T Tani, T Hanafusa, T Nozaki, S Takemoto, ...
Cell reports 2 (6), 1645-1656

### 2011
[Tight associations between transcription promoter type and epigenetic variation in histone positioning and modification](https://link.springer.com/article/10.1186/1471-2164-12-416)  
T Nozaki, N Yachie, R Ogawa, A Kratz, R Saito, M Tomita
BMC genomics 12 (1), 416

[Computational analysis suggests a highly bendable, fragile structure for nucleosomal DNA](https://www.sciencedirect.com/science/article/abs/pii/S0378111911000710)  
T Nozaki, N Yachie, R Ogawa, R Saito, M Tomita
Gene 476 (1-2), 10-14

### Doctoral Dissertation
[Analysis of chromatin structure and dynamics with single nucleosome imaging](https://core.ac.uk/download/pdf/145782733.pdf)  
T Nozaki
